src/Highlight.ts

```diff
@@ -93,11 +91,15 @@ export default class Highlight {
     return this;
   }
 
-  /**
-   * Add class 'focus' to all elements of this highlight.
-   */
+  public updateStartMarker(el: HTMLElement, position: string) {
+    el.dataset.startMessage = this.getMessage(`i18n:highlighter:highlight:${position}`);
+  }
+
   public addFocusedStyles(): Highlight {
-    this.elements.forEach((el: HTMLElement) => el.classList.add(FOCUS_CSS));
+    this.elements.forEach((el: HTMLElement) => {
+      el.setAttribute('aria-current', 'true');
```

Comment:
do you have any literature about `aria-current` ? i don't know much about it but the mdn page makes it seem like it might be meant for a current element in a discrete list, and i'm concerned that our `mark` elements are spread over the whole content and not in a group like that

Reply:
I can take `aria-current` out. (It certainly shouldn't be`aria-selected`.)
The screenreader should distinguish start-selected-highlight vs start-highlight and maybe that's all it needs.

Reply:
I have updated the Highlighter branch to remove `aria-current` and restore the old `focus` class (for CSS selection purposes). A Rex branch for testing it is [here](https://github.com/openstax/rex-web/pull/2242)

Reply:
is it not updated in this PR?

Reply:
Whoops, forgot to push. This PR is updated now.